### PR TITLE
Adds a notice for Crowdin for translations

### LINKED ISSUE #274: Crowdin for translations
Would be nice to use Crowdin for translating the Editor like we do with Opencast

### PR DESCRIPTION
**Pull request by Arnei**
This PR adds a notice about the Crowdin project I set up for this repository. If you are familiar with Crowdin, please check if everything is set up as you would expect it to be.

Resolves #274.

### CODE REVIEW

**Comment by lkiesow** [comment]
It may make sense to put this under the Opencast account:
https://crowdin.com/profile/opencast

**Comment by Arnei** [comment]
The Crowdin project should now belong to the Opencast user.

File: README.md [hunk 1]
```diff
@@ -70,4 +70,6 @@ The editor accesses the following endpoints in Opencast:
 If you want to use current editor frontend with an earlier Opencast version, you will have to cherry pick the relevant commits from the Opencast repository yourself.
 
 
-
+Translating the Editor
+-------------
+You can help translating the editor to your language on crowdin.com/project/opencast-editor. Simply request to join the project on Crowdin and start translating. If you are interested in translating a language which is not a target language right now, please create a Gitub issue and we will add the language.
```
Comment (lkiesow) [inline README.md:75]:
```suggestion
You can help translating the editor to your language on crowdin.com/project/opencast-editor. Simply request to join the project on Crowdin and start translating. If you are interested in translating a language which is not a target language right now, please create a GitHub issue and we will add the language.
```

File: README.md [hunk 2]
```diff
@@ -70,4 +70,6 @@ The editor accesses the following endpoints in Opencast:
 If you want to use current editor frontend with an earlier Opencast version, you will have to cherry pick the relevant commits from the Opencast repository yourself.
 
 
-
+Translating the Editor
+-------------
+You can help translating the editor to your language on crowdin.com/project/opencast-editor. Simply request to join the project on Crowdin and start translating. If you are interested in translating a language which is not a target language right now, please create a GitHub issue and we will add the language.
```
Comment (lkiesow) [inline README.md:75]:
```suggestion
You can help translating the editor to your language on [crowdin.com/project/opencast-editor](https://crowdin.com/project/opencast-editor). Simply request to join the project on Crowdin and start translating. If you are interested in translating a language which is not a target language right now, please create [a GitHub issue](https://github.com/elan-ev/opencast-editor/issues) and we will add the language.
```

**Pull request merged**